packages/types/package.json

```diff
@@ -11,13 +11,16 @@
     "url": "https://github.com/rariblecom/ts-common.git"
   },
   "license": "MIT",
-  "main": "build/index.js",
-  "types": "build/index.d.ts",
+  "main": "build/cjs/index.js",
```

Comment:
You need to change entrypoint. Otherwise for esm - you need to import:
`@rarible/types/build/esm/` instead of `@rarible/types` for cjs

Reply:
<img width="293" alt="image" src="https://github.com/rarible/ts-common/assets/439188/cdd667bf-5afb-4ab5-91fe-2818c81c7b8c">
@Aetet the same way is done in other packages

Reply:
https://www.npmjs.com/package/rxjs?activeTab=code